package.json

```diff
@@ -79,13 +79,9 @@
     "oad": "./dist/cli.js"
   },
   "jest": {
+    "preset": "ts-jest",
```

Comment:
What is the benefit of doing this? Could you share the official doc explaining the benefit of this?

Reply:
https://kulshekhar.github.io/ts-jest/docs/getting-started/installation/#jest-config-file

Additionally, you already have `ts-jest` installed, you were just misusing it.

Basically, this allows Jest to run without a transpilation step. Which improves greatly the devExp as links to certain lines of files go directly to the `.ts` files instead of the `.js` files.